scriptmodules/supplementary/xpadneo.sh

```diff
@@ -12,6 +12,7 @@
 rp_module_id="xpadneo"
 rp_module_desc="Advanced Linux driver for Xbox One wireless gamepads"
 rp_module_licence="GPL3 https://raw.githubusercontent.com/atar-axis/xpadneo/master/LICENSE"
+rp_module_repo="git https://github.com/atar-axis/xpadneo.git 0.9"
```

Comment:
Small typo - the release tag is `v0.9`, not `0.9`.

Reply:
I should have run the check_repos after last changes :-) but it does pick up on this which is good. Thanks.